WordPress/Classes/Models/PostType.h

```diff
@@ -0,0 +1,16 @@
+#import <CoreData/CoreData.h>
+
+@class Blog;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface PostType : NSManagedObject
+
+@property (nullable, nonatomic, retain) NSString *label;
+@property (nullable, nonatomic, retain) NSString *name;
+@property (nullable, nonatomic, retain) NSNumber *apiQueryable;
+@property (nullable, nonatomic, retain) Blog *blog;
```

Comment:
I know they are synonyms, but maybe `strong` instead of `retain` for arc-iness and consistency within the app.